next.config.js

```diff
@@ -4,12 +4,18 @@ const withBundleAnalyzer = require('@next/bundle-analyzer')({
 })
 const { withSentryConfig } = require('@sentry/nextjs')
 
+// @ts-check
+
+/**
+ * @type {import('next').NextConfig}
+ **/
 const config = withBundleAnalyzer({
   async redirects() {
     return [
       {
-        source: '/comparator',
-        destination: '/',
+        source: '/',
+        destination: '/comparator',
+        has: [{ type: 'query', key: 'repo' }],
```

Comment:
This is the definition to redirect from previous `/` to `/comparator` only when necessary (i.e. when the URL has a querystring `repo`). By this config:

- accessing `/` will stay in `/` URL
- accessing `/?repo=renovatebot%2Frenovate&from=25.54.0&to=latest` will redirect to `/comparator?repo=renovatebot%2Frenovate&from=25.54.0&to=latest` URL
- accessing `/?repo=renovatebot%2Frenovate` will redirect to `/comparator?repo=renovatebot%2Frenovate` URL
- accessing `/?foo=bar` will stay in `/?foo=bar`

Reply:
I've tested this and it seems to work well! 😄